clang/lib/CIR/CodeGen/CIRGenStmt.cpp

```diff
@@ -80,12 +80,12 @@ mlir::LogicalResult CIRGenFunction::emitStmt(const Stmt *s,
 #include "clang/AST/StmtNodes.inc"
     {
       // Remember the block we came in on.
-      mlir::Block *incoming = builder.getInsertionBlock();
+      [[maybe_unused]] mlir::Block *incoming = builder.getInsertionBlock();
```

Comment:
Classic codegen uses these to erase the outgoing block if it doesn't match the incoming block and isn't used. The incubator doesn't do that, perhaps because the canonicalize pass will erase it.